app/src/main/kotlin/no/nav/k9punsj/innsending/dto/NyJournalpost.kt

```diff
@@ -0,0 +1,63 @@
+package no.nav.k9punsj.innsending.dto
+
+import com.fasterxml.jackson.databind.node.ObjectNode
+import no.nav.k9punsj.felles.Identitetsnummer
+import org.intellij.lang.annotations.Language
+import java.util.*
+
+internal data class NyJournalpost(
+    internal val correlationId: String,
+    internal val tittel: String,
+    internal val brevkode: String,
+    internal val fagsystem: String,
+    internal val saksnummer: String,
+    internal val identitetsnummer: Identitetsnummer,
+    internal val avsenderNavn: String,
+    internal val pdf: ByteArray,
+    internal val json: ObjectNode) {
+
+    internal fun dokarkivPayload() : String {
+        @Language("JSON")
+        val json = """
+            {
+              "eksternReferanseId": "$correlationId",
+              "tittel": "$tittel",
+              "avsenderMottaker": {
+                "navn": "$avsenderNavn"
+              },
+              "bruker": {
+                "id": "$identitetsnummer",
+                "idType": "FNR"
+              },
+              "sak": {
+                "sakstype": "FAGSAK",
+                "fagsakId": "$saksnummer",
+                "fagsaksystem": "$fagsystem"
+              },
+              "dokumenter": [{
+                "tittel": "$tittel",
+                "brevkode": "$brevkode",
+                "dokumentVarianter": [{
+                  "filtype": "PDFA",
+                  "variantformat": "ARKIV",
+                  "fysiskDokument": "${pdf.base64()}"
+                },{
+                  "filtype": "JSON",
+                  "variantformat": "ORIGINAL",
+                  "fysiskDokument": "${json.base64()}"
+                }]
+              }],
+              "tema": "OMS",
+              "journalposttype": "NOTAT",
```

Comment:
Er det riktig at `journalposttypen` her skal være `NOTAT`

Reply:
Ja:
`NOTAT brukes for dokumentasjon som NAV har produsert selv og uten mål om å distribuere dette ut av NAV. Eksempler på dette er forvaltningsnotater og referater fra telefonsamtaler med brukere.`

Reply:
Vil det si at alle nye journalposter man oppretter via punsj skal være journalpostnotater?

Reply:
Yes

Reply:
Journalposttype Utgående er jo effektivt ett brev och skall håndteres av formidling.